.ci/docker/requirements-ci.txt

```diff
@@ -295,13 +295,19 @@ tensorboard==2.13.0
 #Pinned versions:
 #test that import: test_tensorboard
 
+tomli ; python_version < "3.11"
+#Desription: This is a requirement for Executorch
+
 pywavelets==1.4.1 ; python_version < "3.12"
 pywavelets==1.5.0 ; python_version >= "3.12"
 #Description: This is a requirement of scikit-image, we need to pin
 # it here because 1.5.0 conflicts with numpy 1.21.2 used in CI
 #Pinned versions: 1.4.1
 #test that import:
 
+zstd
+#Description: This is a requirement for Executorch
+
```

Comment:
I don't think these changes are needed.
From the log, the file that is used is from executorch https://github.com/pytorch/executorch/blob/main/.ci/docker/requirements-ci.txt

I'm not sure why the build is failing though.